### ISSUE TITLE
hockey action fails

### ISSUE THREAD

**Issue by grzegorzkrukowski** [fastlane/fastlane#9640]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

`hockey` action fails on newest fastlane version
`Unknown method 'response'`
`To call another action from an action use 'other_action.response' instead`

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

+------------------------------------------------------------------------------+

| (2) fastlane@2.3.3                                                           |
+------------------------------------------------------------------------------+
| id: fastlane                                                                 |
| version: 2.3.3                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2017-06-30T08:36:47-07:00                                              |
+------------------------------------------------------------------------------+

[08:56:34]: --------------------
[08:56:34]: --- Step: hockey ---
[08:56:34]: --------------------
[08:56:34]: Starting with file(s) upload to HockeyApp... this could take some time.
[08:56:53]: Unknown method 'response'
[08:56:53]: Sending crash report...
[08:56:53]: The stacktrace is sanitized so no personal information is sent.
[08:56:53]: To see what we are sending, look here: /Users/vagrant/.fastlane/latest_crash.json
[08:56:53]: Learn more at https://github.com/fastlane/fastlane#crash-reporting
[08:56:53]: You can disable crash reporting by adding `opt_out_crash_reporting` at the top of your Fastfile
[08:56:54]: To call another action from an action use `other_action.response` instead

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

### Stack

| Key                         | Value                                            |
| --------------------------- | ------------------------------------------------ |
| OS                          | 10.12.5                                          |
| Ruby                        | 2.4.0                                            |
| Bundler?                    | false                                            |
| Git                         | git version 2.11.0 (Apple Git-81)                |
| Installation Source         | /usr/local/bin/fastlane                          |
| Host                        | Mac OS X 10.12.5 (16F73)                         |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.0/lib                 |
| OpenSSL Version             | OpenSSL 1.0.2j  26 Sep 2016                      |
| Is contained                | false                                            |
| Is homebrew                 | false                                            |
| Is installed via Fabric.app | false                                            |
| Xcode Path                  | /Applications/Xcode8.3.3.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                            |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`
# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
# fastlane_version "2.9.0"

default_platform :ios

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby

### Variables

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.43.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 1.5.1        |
| claide                    | 1.0.1        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.4        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 2.1.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| multi_json                | 1.12.1       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.12.0       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| bundler                   | 1.14.3       |
| faraday_middleware        | 0.11.0.1     |
| json                      | 2.1.0        |
| io-console                | 0.4.6        |
| excon                     | 0.56.0       |
| openssl                   | 2.0.2        |
| rubyzip                   | 1.2.1        |
| plist                     | 3.3.0        |
| faraday                   | 0.12.1       |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| fastimage                 | 2.1.0        |
| terminal-table            | 1.8.0        |
| unicode-display_width     | 1.2.1        |
</details>

*generated on:* **2017-06-30**
</details>

**Comment by PetrosIchor** [comment]
How are you running hockey?  You're Fastfile shows empty on my end.

**Comment by KrauseFx** [comment]
Yeah, please update to _fastlane_ 2.43.1, where this issue should be fixed 👍  Sorry about the regression
